lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,33 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
+    <%- edx.StringUtils.interpolate(
           gettext("You still need to visit the %(display_name)s website to complete the credit process."),
           { display_name: display_name }, true
         ) %>
   </div>
   <div class="provider-more-info">
-    <%= interpolate(
+    <%- edx.StringUtils.interpolate(
           gettext("To finalize course credit, %(display_name)s requires %(platform_name)s learners to submit a credit request."),
           { display_name: display_name, platform_name: platformName }, true
         ) %>
   </div>
   <div class="provider-instructions">
-    <%= fulfillment_instructions %>
+    <%- fulfillment_instructions %>
   </div>
 </div>
 
 <div class="provider-buttons-logos">
   <div class="provider-logo">
-    <%= interpolate("<img src='%s' alt='%s'></image>", [thumbnail_url, display_name]) %>
+    <%- edx.HtmlUtils.interpolateHtml(
+                edx.HtmlUtils.HTML('<img src="{thumbnailUrl}" alt="{displayName}"></image>'),
+                {thumbnailUrl:thumbnail_url ,displayName:display_name}
+                 )
+                 %>
```

Comment:
You should use <%= .. %>  to avoid double escaping. But still you are not escaping displayName, thumbnail_url. You have to see how to escape them too or is it required
```
<%= edx.HtmlUtils.interpolateHtml(
                edx.HtmlUtils.HTML('<img src="{thumbnailUrl}" alt="{displayName}"></image>'),
                {thumbnailUrl:thumbnail_url ,displayName:display_name}
                 )
                 %>
```

Reply:
I don't think displayName need to be escaped because edx.HtmlUtils.HTML would takecare of that.

Reply:
Nope. By calling HTML() just mean that whatever iam passing is html safe so don't escape it. So one thing is correct we don't want image tag to be escaped but we also don't want displayName/thumbnail_url to remain unescape.

Reply:
Okay you don't need to do this change. HtmlUtils.interpolateHtml function is doing this for us.